ftests/999-stress-cgroup_init.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Stressing cgroup_int() code
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.  All rights reserved.
+# Author: Kamalesh Babulal <kamalesh.babulal@oracle.com>
+#
+
+from libcgroup import Cgroup
+from run import Run
+import consts
+import ftests
+import sys
+import os
+
+MNT_COUNT = 101
+MNT_POINT = '/tmp/999stress/'
+DIR_PREFIX = 'name'
+
+
+def prereqs(config):
+    return consts.TEST_PASSED, None
+
+
+def setup(config):
+    cmd = ['sudo', 'mkdir']
+
+    cmd.append(MNT_POINT)
+
+    for count in range(MNT_COUNT):
+        cmd.append(MNT_POINT + DIR_PREFIX + str(count))
```

Comment:
One nitpick - `MNT_POINT + DIR_PREFIX + str(count)` is copy/pasted several times throughout this test.  Should we hide this inside of a helper function?

Reply:
@drakenclimber Thanks for the review.  Good idea, let me re-spin the test case with a helper function for building the cgroup path.